lib/sync-srv.js

```diff
@@ -336,32 +350,41 @@ function processPending(dataset_id, dataset, params, cb) {
           syncUtil.doLog(dataset_id, 'verbose', ' READ for DELETE Success', params);
           syncUtil.doLog(dataset_id, 'silly', ' READ for DELETE Data : \n' + util.inspect(data), params);
 
-          var preHash = syncUtil.generateHash(pre);
-          var dataHash = syncUtil.generateHash(data);
+          //var preHash = syncUtil.generateHash(pre);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ACommentedCode)